### PR TITLE
Metadata review - wfp FLORTK FLORDL

### PR DESCRIPTION
**Pull request by johnmlund**
Updates to WFP FLORTK and FLORDL cal csv per Metadata review project.

### CODE REVIEW

File: calibration/FLORDL/CGINS-FLORDL-02350__20121120.csv [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
-2350,CC_dark_counts_chlorophyll_a,47.0,
-2350,CC_dark_counts_volume_scatter,48.0,Requires PD1960/1962 (tempwat/pracsal) from nearby CTD (GI02HYPM-WFP02-04-CTDPFL000?)
+2350,CC_dark_counts_chlorophyll_a,48.0,
+2350,CC_dark_counts_volume_scatter,48.0,
 2350,CC_depolarization_ratio,0.039,Constant
 2350,CC_measurement_wavelength,700.0,Constant
 2350,CC_scale_factor_chlorophyll_a,0.0073,
-2350,CC_scale_factor_volume_scatter,2.005e-06,
+2350,CC_scale_factor_volume_scatter,2.034e-06,
 2350,CC_scattering_angle,140.0,Constant
```
Comment (akjensen7) [inline calibration/FLORDL/CGINS-FLORDL-02350__20121120.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-02350__20161019.csv [hunk 2]
```diff
@@ -5,5 +5,5 @@ serial,name,value,notes
 2350,CC_depolarization_ratio,0.039,Constant
 2350,CC_measurement_wavelength,700,Constant
 2350,CC_scale_factor_chlorophyll_a,0.0071,
-2350,CC_scale_factor_volume_scatter,1.93E-06,
+2350,CC_scale_factor_volume_scatter,1.929E-06,
 2350,CC_scattering_angle,140,Constant
```
Comment (akjensen7) [inline calibration/FLORDL/CGINS-FLORDL-02350__20161019.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-02350__20181107.csv [hunk 3]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+2350,CC_angular_resolution,1.096,Constant
+2350,CC_dark_counts_chlorophyll_a,50,
+2350,CC_dark_counts_volume_scatter,46,
+2350,CC_depolarization_ratio,0.039,Constant
+2350,CC_measurement_wavelength,700,Constant
+2350,CC_scale_factor_chlorophyll_a,0.0070,
+2350,CC_scale_factor_volume_scatter,1.765E-06,
+2350,CC_scattering_angle,140,Constant
```
Comment (akjensen7) [inline calibration/FLORDL/CGINS-FLORDL-02350__20181107.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-02736__20120713.csv [hunk 4]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+2736,CC_angular_resolution,1.096,Constant
+2736,CC_dark_counts_chlorophyll_a,52.0,
+2736,CC_dark_counts_volume_scatter,51.0,
+2736,CC_depolarization_ratio,0.039,Constant
+2736,CC_measurement_wavelength,700.0,Constant
+2736,CC_scale_factor_chlorophyll_a,0.0073,
+2736,CC_scale_factor_volume_scatter,1.871e-06,
+2736,CC_scattering_angle,140.0,Constant
```
Comment (akjensen7) [inline calibration/FLORDL/CGINS-FLORDL-02736__20120713.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-02934__20161019.csv [hunk 5]
```diff
@@ -5,5 +5,5 @@ serial,name,value,notes
 2934,CC_depolarization_ratio,0.039,Constant
 2934,CC_measurement_wavelength,700,Constant
 2934,CC_scale_factor_chlorophyll_a,0.007,
-2934,CC_scale_factor_volume_scatter,1.66E-06,
+2934,CC_scale_factor_volume_scatter,1.659E-06,
 2934,CC_scattering_angle,140,Constant
```
Comment (akjensen7) [inline calibration/FLORDL/CGINS-FLORDL-02934__20161019.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-02934__20181107.csv [hunk 6]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+2934,CC_angular_resolution,1.096,Constant
+2934,CC_dark_counts_chlorophyll_a,50,
+2934,CC_dark_counts_volume_scatter,44,
+2934,CC_depolarization_ratio,0.039,Constant
+2934,CC_measurement_wavelength,700,Constant
+2934,CC_scale_factor_chlorophyll_a,0.0069,
+2934,CC_scale_factor_volume_scatter,1.625E-06,
+2934,CC_scattering_angle,140,Constant
```
Comment (akjensen7) [inline calibration/FLORDL/CGINS-FLORDL-02934__20181107.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-03280__20171118.csv [hunk 7]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+3280,CC_angular_resolution,1.096,Constant
+3280,CC_dark_counts_chlorophyll_a,50,
+3280,CC_dark_counts_volume_scatter,47,
+3280,CC_depolarization_ratio,0.039,Constant
+3280,CC_measurement_wavelength,700.0,Constant
+3280,CC_scale_factor_chlorophyll_a,0.0067,
+3280,CC_scale_factor_volume_scatter,1.777e-06,
+3280,CC_scattering_angle,140.0,Constant
```
Comment (akjensen7) [inline calibration/FLORDL/CGINS-FLORDL-03280__20171118.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-03284__20130828.csv [hunk 8]
```diff
@@ -1,9 +1,9 @@
...
-3284,CC_dark_counts_chlorophyll_a,48.0,
-3284,CC_dark_counts_volume_scatter,48.0,Requires PD1960/1962 (tempwat/pracsal) from nearby CTD (GP02HYPM-WFP03-04-CTDPFL000?)
+3284,CC_dark_counts_chlorophyll_a,51,
+3284,CC_dark_counts_volume_scatter,50,
 3284,CC_depolarization_ratio,0.039,Constant
 3284,CC_measurement_wavelength,700.0,Constant
 3284,CC_scale_factor_chlorophyll_a,0.0073,
-3284,CC_scale_factor_volume_scatter,2.034e-06,
+3284,CC_scale_factor_volume_scatter,1.713e-06,
 3284,CC_scattering_angle,140.0,Constant
```
Comment (akjensen7) [inline calibration/FLORDL/CGINS-FLORDL-03284__20130828.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-03284__20161017.csv [hunk 9]
```diff
@@ -5,5 +5,5 @@ serial,name,value,notes
 3284,CC_depolarization_ratio,0.039,Constant
 3284,CC_measurement_wavelength,700,Constant
 3284,CC_scale_factor_chlorophyll_a,0.0072,
-3284,CC_scale_factor_volume_scatter,1.82E-06,
+3284,CC_scale_factor_volume_scatter,1.823E-06,
 3284,CC_scattering_angle,140,Constant
```
Comment (akjensen7) [inline calibration/FLORDL/CGINS-FLORDL-03284__20161017.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-03284__20181107.csv [hunk 10]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+3284,CC_angular_resolution,1.096,Constant
+3284,CC_dark_counts_chlorophyll_a,52,
+3284,CC_dark_counts_volume_scatter,46,
+3284,CC_depolarization_ratio,0.039,Constant
+3284,CC_measurement_wavelength,700,Constant
+3284,CC_scale_factor_chlorophyll_a,0.0070,
+3284,CC_scale_factor_volume_scatter,1.715E-06,
+3284,CC_scattering_angle,140,Constant
```
Comment (akjensen7) [inline calibration/FLORDL/CGINS-FLORDL-03284__20181107.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-02350__20121120.csv [hunk 11]
```diff
@@ -1,9 +1,9 @@
...
 2350,CC_angular_resolution,1.096,Constant
-2350,CC_dark_counts_chlorophyll_a,47.0,
-2350,CC_dark_counts_volume_scatter,48.0,Requires PD1960/1962 (tempwat/pracsal) from nearby CTD (GI02HYPM-WFP02-04-CTDPFL000?)
+2350,CC_dark_counts_chlorophyll_a,48.0,
+2350,CC_dark_counts_volume_scatter,48.0,
 2350,CC_depolarization_ratio,0.039,Constant
 2350,CC_measurement_wavelength,700.0,Constant
 2350,CC_scale_factor_chlorophyll_a,0.0073,
-2350,CC_scale_factor_volume_scatter,2.005e-06,
+2350,CC_scale_factor_volume_scatter,2.034e-06,
```
Comment (reedan88) [inline calibration/FLORDL/CGINS-FLORDL-02350__20121120.csv]: Looks good

File: calibration/FLORDL/CGINS-FLORDL-02350__20161019.csv (hunk 2 above)
Comment (reedan88) [inline calibration/FLORDL/CGINS-FLORDL-02350__20161019.csv]: Looks good

File: calibration/FLORDL/CGINS-FLORDL-02350__20181107.csv (hunk 3 above)
Comment (reedan88) [inline calibration/FLORDL/CGINS-FLORDL-02350__20181107.csv]: Looks good

File: calibration/FLORDL/CGINS-FLORDL-02736__20120713.csv (hunk 4 above)
Comment (reedan88) [inline calibration/FLORDL/CGINS-FLORDL-02736__20120713.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-02934__20161019.csv (hunk 5 above)
Comment (reedan88) [inline calibration/FLORDL/CGINS-FLORDL-02934__20161019.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-02934__20181107.csv (hunk 6 above)
Comment (reedan88) [inline calibration/FLORDL/CGINS-FLORDL-02934__20181107.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-03280__20171118.csv (hunk 7 above)
Comment (reedan88) [inline calibration/FLORDL/CGINS-FLORDL-03280__20171118.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-03284__20130828.csv (hunk 8 above)
Comment (reedan88) [inline calibration/FLORDL/CGINS-FLORDL-03284__20130828.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-03284__20161017.csv (hunk 9 above)
Comment (reedan88) [inline calibration/FLORDL/CGINS-FLORDL-03284__20161017.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-03284__20181107.csv (hunk 10 above)
Comment (reedan88) [inline calibration/FLORDL/CGINS-FLORDL-03284__20181107.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-04013__20150629.csv [hunk 12]
```diff
@@ -1,9 +1,9 @@
...
 4013,CC_angular_resolution,1.096,Constant
 4013,CC_dark_counts_chlorophyll_a,46.0,
-4013,CC_dark_counts_volume_scatter,44.0,Requires PD1960/1962 (tempwat/pracsal) from nearby CTD (GA02HYPM-WFP02-04-CTDPFL000?)
+4013,CC_dark_counts_volume_scatter,44.0,
 4013,CC_depolarization_ratio,0.039,Constant
 4013,CC_measurement_wavelength,700.0,Constant
 4013,CC_scale_factor_chlorophyll_a,0.0073,
 4013,CC_scale_factor_volume_scatter,1.648e-06,
-4013,CC_scattering_angle,140.0,24
+4013,CC_scattering_angle,140.0,Constant
```
Comment (reedan88) [inline calibration/FLORDL/CGINS-FLORDL-04013__20150629.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-04013__20170509.csv [hunk 13]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+4013,CC_angular_resolution,1.096,Constant
+4013,CC_dark_counts_chlorophyll_a,50,
+4013,CC_dark_counts_volume_scatter,47,
+4013,CC_depolarization_ratio,0.039,Constant
+4013,CC_measurement_wavelength,700.0,Constant
+4013,CC_scale_factor_chlorophyll_a,0.0070,
+4013,CC_scale_factor_volume_scatter,1.684e-06,
+4013,CC_scattering_angle,140.0,Constant
```
Comment (reedan88) [inline calibration/FLORDL/CGINS-FLORDL-04013__20170509.csv]: Looks good.

File: calibration/FLORDL/CGINS-FLORDL-04014__20170509.csv [hunk 14]
```diff
@@ -0,0 +1,9 @@
+serial,name,value,notes
+4014,CC_angular_resolution,1.096,Constant
+4014,CC_dark_counts_chlorophyll_a,50,
+4014,CC_dark_counts_volume_scatter,47,
+4014,CC_depolarization_ratio,0.039,Constant
+4014,CC_measurement_wavelength,700.0,Constant
+4014,CC_scale_factor_chlorophyll_a,0.0071,
+4014,CC_scale_factor_volume_scatter,1.648e-06,
+4014,CC_scattering_angle,140.0,Constant
```
Comment (reedan88) [inline calibration/FLORDL/CGINS-FLORDL-04014__20170509.csv]: Looks good.

File: calibration/FLORTK/CGINS-FLORTK-01029__20140814.csv [hunk 15]
```diff
@@ -5,7 +5,7 @@ serial,name,value,notes
 1029,CC_dark_counts_volume_scatter,49.0,
 1029,CC_depolarization_ratio,0.039,
 1029,CC_measurement_wavelength,700.0,
-1029,CC_scale_factor_cdom,0.07877,
+1029,CC_scale_factor_cdom,0.0787,
 1029,CC_scale_factor_chlorophyll_a,0.0117,
 1029,CC_scale_factor_volume_scatter,3.338e-06,
 1029,CC_scattering_angle,124.0,
```
Comment (reedan88) [inline calibration/FLORTK/CGINS-FLORTK-01029__20140814.csv]: Looks good.

File: calibration/FLORTK/CGINS-FLORTK-01119__20130927.csv [hunk 16]
```diff
@@ -1,11 +1,9 @@
...
+1119,CC_angular_resolution,1.076,
+1119,CC_dark_counts_cdom,60.0,
 1119,CC_dark_counts_chlorophyll_a,50.0,
 1119,CC_dark_counts_volume_scatter,49.0,
-1119,CC_depolarization_ratio,0.039,"Default values defined in DPA: def flo_scat_seawater(degC, psu, theta=117.0, wlngth=700.0, delta=0.039)
-"
+1119,CC_depolarization_ratio,0.039,
 1119,CC_measurement_wavelength,700.0,
 1119,CC_scale_factor_cdom,0.0904,
 1119,CC_scale_factor_chlorophyll_a,0.0125,
```
Comment (reedan88) [inline calibration/FLORTK/CGINS-FLORTK-01119__20130927.csv]: Looks good.

**Comment by rtravis-whoi** [comment]
filename fixed in Vault per Andrew's review comment.

**Pull request merged**